src/xSTIR/cSTIR/cstir.cpp

```diff
@@ -417,6 +417,23 @@ void* cSTIR_objectFromFile(const char* name, const char* filename)
 	CATCH;
 }
 
+typedef xSTIR_PoissonLLhLinModMeanListDataProjMatBin3DF LMObjFun;
+
+extern "C"
+void* cSTIR_objFunListModeSetInterval(void* ptr_f, size_t ptr_data)
```

Comment:
Not really for a question for this PR, but why `size_t ptr_data` and not `void * ptr_data`? On some systems, they are not the same size, and this could therefore create trouble.

Reply:
I vaguely remember having some SWIG trouble with `void*` but no longer remember what it was. Perhaps with the latest SWIG `void*` is now ok.

Reply:
BTW SWIG actually still does not like passing the numpy array data pointer as `void*`  - tried with `cSTIR_getImageData`, got this:
```
Traceback (most recent call last):
  File "/home/sirfuser/devel/buildVM/sources/SIRF/examples/Python/PET/acquisition_data.py", line 172, in <module>
    main()
  File "/home/sirfuser/devel/buildVM/sources/SIRF/examples/Python/PET/acquisition_data.py", line 141, in main
    image_array = image.as_array()
  File "/home/sirfuser/devel/install/python/sirf/STIR.py", line 611, in as_array
    try_calling(pystir.cSTIR_getImageData(self.handle, array.ctypes.data))
  File "/home/sirfuser/devel/install/python/sirf/pystir.py", line 306, in cSTIR_getImageData
    return _pystir.cSTIR_getImageData(ptr, ptr_data)
TypeError: in method 'cSTIR_getImageData', argument 2 of type 'void *'
sirfuser@vagrant:~/devel/buildVM/sources/SIRF/examples/Python/PET$ 
```
same story with `float*`:
```
Traceback (most recent call last):
  File "/home/sirfuser/devel/buildVM/sources/SIRF/examples/Python/PET/acquisition_data.py", line 172, in <module>
    main()
  File "/home/sirfuser/devel/buildVM/sources/SIRF/examples/Python/PET/acquisition_data.py", line 141, in main
    image_array = image.as_array()
  File "/home/sirfuser/devel/install/python/sirf/STIR.py", line 611, in as_array
    try_calling(pystir.cSTIR_getImageData(self.handle, array.ctypes.data))
  File "/home/sirfuser/devel/install/python/sirf/pystir.py", line 306, in cSTIR_getImageData
    return _pystir.cSTIR_getImageData(ptr, ptr_data)
TypeError: in method 'cSTIR_getImageData', argument 2 of type 'float *'
```

Reply:
hmmm. weird. Here's an example that does that https://stackoverflow.com/a/37308401. Anyway, let's leave that for later!